finetune/config.py

```diff
@@ -272,7 +276,7 @@ def get_default_config():
         n_layer=12,
         act_fn="gelu",
         n_embed=768,
```

Comment:
What do you think about having n_heads, n_layers and n_embed set to None as default, and add defaults into the base save files?